docs/source/deploy_pws.md

```diff
@@ -0,0 +1,141 @@
+# Deployment Scripts for Pivotal Web Services
```

Comment:
We haven't tried PWS in a long time. Maybe we should remove this file.

Reply:
Removed.